stack.yaml

```diff
@@ -38,7 +38,9 @@ packages:
 # Dependency packages to be pulled from upstream that are not in the resolver
 # using the same syntax as the packages field.
 # (e.g., acme-missiles-0.3)
-# extra-deps: []
+extra-deps:
+  - git: "https://github.com/taruti/haskell-hex"
+    commit: 5953ae445ae8f608858903280fe3d9d2b503ae31 
```

Comment:
Don't we need the sha-256 in comment nowadays?

Reply:
When you say in comments, you mean in double quotes ? Not that I know of. At the very least, this builds, but if you have a more fashionable way of writing this, I'm more than willing to do it !

Reply:
(However the quotes are not needed in the `git` value, I should remove these)

Reply:
I might be mixing things up. I had the vague memory of a yaml comment to hint stack about the sha256 of a tier library...

Reply:
Ah yes, but that's for a `hackage` release (you're most likely thinking about https://docs.haskellstack.org/en/stable/pantry/#hackage), `stack` would have told me if that was an option when I tried to build.

Reply:
That's right

Reply:
That's right.